### DISCUSSION TITLE
WHy are there 2 checkboxes for "Retract On Layer Change" and Wipe While Retracting"

### DISCUSSION THREAD

**Discussion by Aurasphere-au** [OrcaSlicer/OrcaSlicer#6959] [General]
Im sure there is a reason but it doesnt follow logic

![image](https://github.com/user-attachments/assets/1ca1a150-f537-454b-be10-9a4015b75124)

**Comment by LJKraus** [comment]
There are two boxes because they are two different functions.
![image](https://github.com/user-attachments/assets/7bac0d64-db36-4779-bff5-34909a227f5d)
![image](https://github.com/user-attachments/assets/0ad43957-9db4-4303-9db8-9c7d704a042c)
You may or may not want to retract on a layer change.
You may or may not want to wipe with every retraction.

You can choose whatever combination you prefer.

**Reply by Aurasphere-au** [reply]
Hmm thanks for the reply but I think you misread the title
Why are there 2 check boxes on each of those options. From a UX point of view, having 2 UI widgets for the same thing is well...strange and useless so Im guess there might be another reason beyond the obvious or its a mistake
ie You can actually select each check box individually
![image](https://github.com/user-attachments/assets/d7554836-be84-417d-81c5-d484f29fdbe0)
![image](https://github.com/user-attachments/assets/0c490051-f28d-48a3-9ac0-b041f595494a)

**Comment by LJKraus** [comment]
You are right, I missed the boxes on the right.    There is a another instance of this for  "Long retraction when cut".

The box on the right cannot be selected unless the box on the left also has a check mark.  These three options are the the only ones that are either on or off, and would each require both checks to be active. 

That is consistent with the other overrides that require a check on the left to activate the option.    The other overrides need a left check to make them active and then a choice from multiple options or values.  I think the advantage in those cases is that an override can be temporarily turned off without losing the customized value.  That is not much use for a binary setting.
